src/api/Api.ts

```diff
@@ -62,13 +62,17 @@ export class ApiImpl implements Api {
     async info(): Promise<ApiInfo> {
         const response = await fetch(this.getRoutingURLWithKey('info').toString(), {
             headers: { Accept: 'application/json' },
-        })
+        }).catch(() => { throw new Error('Could not connect to the Service. Try to reload!') });
```

Comment:
Any network issues where requests cannot be sent or responses cannot be received should be catched here, and "Could not connect to the Service" pretty much sums this up.